docs/hugo/content/guide/keyvaults/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "KeyVaults"
+linktitle: "KeyVaults"
+---
+
+The standard options for `createMode` when creating a new KeyVault assume you know whether there is an existing soft-deleted KeyVault already present.
+
+* `default` - create a new KeyVault. Deployment will fail if an existing soft-deleted KeyVault has the same name.
+* `recover` - recover an existing soft-deleted KeyVault. Deployment will fail if no soft-deleted KeyVault is found.
+
+These options works perfectly well when deploying via ARM or Bicep as those deployments are single use.
+
+However, with a goal seeking system that expects to maintain resources over time, these options don't work so well (see _Motivation_, below), so in Azure Service Operator (ASO) v2.4.0 we are introducing two additional options:
+
+* `createOrRecover` - create a new key KeyVault. If an existing soft-deleted KeyVault has the same name, recover and reuse it.
+
+* `purgeThenCreate` - If an existing soft-deleted KeyVault has the same name, purge it first; then create a brand new one. **Dangerous**
```

Comment:
I'd suggest to elaborate more about the `danger` here and remove **Dangerous**

Reply:
There's elaboration about the dangers in the next paragraph. I've made that more explicit.